google-chart.html

```diff
@@ -52,6 +52,7 @@
 <dom-module id="google-chart">
   <link rel="import" type="css" href="google-chart.css">
   <template>
+    <google-chart-loader id="loader" type="[[type]]"></google-chart-loader>
```

Comment:
Hmm, type is technically always set as the default is 'column'.
I think behavior when type undefined is currently undefined.